src/app/archive/create_schema.sql

```diff
@@ -38,15 +38,18 @@ CREATE TABLE tokens
 , owner_token_id       int               REFERENCES tokens(id)
 );
 
-CREATE INDEX idx_tokens_id ON tokens(id);
 CREATE INDEX idx_tokens_value ON tokens(value);
 
+/* insert default token, whose value is obtained via
+   the OCaml code `Token_id.(to_string default)`
+*/
+INSERT INTO tokens (value) VALUES ('wSHV2S4qX9jFsLjQo8r1BsMLH2ZRKsZx6EJd1sbozGPieEC4Jf');
```

Comment:
can we do this as part of inserting genesis data in the [archive process](https://github.com/MinaProtocol/mina/blob/2ff92dd40f10f676a9491d8085ac75e156698044/src/app/archive/archive_lib/processor.ml#L3322) (even if there isn't a config file passed)?